packages/cli/src/x/actions/download.action.ts

```diff
@@ -39,19 +47,23 @@ export async function downloadAction(
       return
     }
 
-    const matchingAgents = agents.filter(agent => agent.slug === slug)
+    const agent = agents.find((agent) => {
+      if (slug) {
+        return agent.slug === slug
+      } else if (agentId) {
+        return agent.id === agentId
+      }
+    })
     
-    if (matchingAgents.length === 0) {
-      spinner.fail(`No agent found with slug: ${slug}`)
+    if (!agent) {
+      spinner.fail(`No agent found with ${slug ? `slug: ${slug}` : `id: ${agentId}`}`)
       return;
     }
 
-    const agent = matchingAgents[0];
     spinner.succeed(`Found agent: ${agent.slug}`)
 
-    // Set exportType to SOURCE by default, only use AUTOBUILD_INLINED if explicitly passed
-    const exportType = options?.exportType || 'SOURCE';
-    
+    const exportType = (agent as any)?.config?.autobuildId ? 'SYSTEM_COMBINED' : 'SOURCE';
```

Comment:
I think this is probably the right path since if you're downloading the autobuild agent, then it probably make most sense to download the SYSTEM_COMBINED, but if we're not going to accept user input, then do we need to get rid of the `exportType` option on line 18?

Reply:
Yes, ty!